gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -372,6 +411,30 @@ MM_VerboseHandlerOutputStandard::handleCompactEndInternal(MM_EnvironmentBase* en
 
 #endif /* defined(OMR_GC_MODRON_COMPACTION) */
 
+void
+MM_VerboseHandlerOutputStandard::handleConcurrentMarkEnd(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
+{
+	MM_ConcurrentPhaseEndEvent *event = (MM_ConcurrentPhaseEndEvent *)eventData;
+	MM_ConcurrentMarkPhaseStats *stats = (MM_ConcurrentMarkPhaseStats *)event->concurrentStats;
+	MM_ConcurrentGCStats *collectionStats = stats->_collectionStats;
+	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+	MM_VerboseWriterChain* writer = _manager->getWriterChain();
+
+	uint64_t duration = 0;
+	bool deltaTimeSuccess = getTimeDeltaInMicroSeconds(&duration, stats->_startTime, stats->_endTime);
+
+	handleGCOPOuterStanzaStart(env, "trace", stats->_cycleID, duration, deltaTimeSuccess);
+	writer->formatAndOutput(env, 1, "<trace bytesTraced=\"%zu\" workStackOverflowCount=\"%zu\" />", collectionStats->getConHelperTraceSizeCount() + collectionStats->getTraceSizeCount(), collectionStats->getConcurrentWorkStackOverflowCount());
+	if (0 == stats->_cardTableStats->getConcurrentCleanedCards()) {
+		writer->formatAndOutput(env, 1, "<card-cleaning bytesTraced=\"%zu\" cardsCleaned=\"%zu\" />", collectionStats->getConHelperCardCleanCount() + collectionStats->getCardCleanCount(), stats->_cardTableStats->getConcurrentCleanedCards());
```

Comment:
Brackets around `collectionStats->getConHelperCardCleanCount() + collectionStats->getCardCleanCount()`

And one L432